### PR TITLE
Molecule: Form item

### PR DESCRIPTION
**Pull request by kevin-wangg**
Will revert DonorHomepage.tsx before merging.

https://user-images.githubusercontent.com/34655694/114311674-b587bb80-9abd-11eb-9197-40c95e38b127.mp4

### CODE REVIEW

File: client/src/components/molecules/FormItem.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+const FormItem: FunctionComponent<Props> = (props: Props) => {
+    const isError = props.errorString.length > 0
+
+    return (
+        <div className="form-item">
+            <div className="form-item-top">
+                <span className={props.isDisabled ? "form-item-disabled" : undefined}>
+                    {props.formItemName}
+                    {props.tooltipText ? <Tooltip tooltipText={props.tooltipText}/> : null}
```
Comment (guy-ro) [inline client/src/components/molecules/FormItem.tsx]: Nitpick, replace with

`{props.tooltipText && <Tooltip tooltipText={props.tooltipText}/>}`

File: client/src/pages/DonorHomepage.tsx [hunk 2]
```diff
@@ -6,10 +6,28 @@ import DonorPage from '../components/layouts/DonorPage'
 import DonorRequestGroupBrowser from '../components/organisms/DonorRequestGroupBrowser'
 import HomepageBanner from '../components/organisms/HomepageBanner'
 
+import FormItem from '../components/molecules/FormItem'
+
+interface Props {
+  isDisabled: boolean;
+}
+const Bob: FunctionComponent<Props> = (props: Props) => {
```
Comment (guy-ro) [inline client/src/pages/DonorHomepage.tsx:14]: hi Bob

**Review by guy-ro — APPROVED** [review]
LGTM!

**Comment by bonnie-chin** [comment]
Ouuu damn wait Megan told me about this yesterday I think mayhaps we gotta make sure the form input is tall enough bc I know there's other instances with the drop down where it gets taller bc the tags are fitted inside

👀 can I ask what the height is? I think it's supposed to be like 45px but maybe the proportions are just tripping me up

**Review by meganniu — APPROVED** [review]
beautiful
perfect

**Pull request merged**